README.md

```diff
@@ -0,0 +1,6 @@
+# .github
+Hello there!
+
+Questo è un repo per l'organizzazione di giornate di smart working a Milano (penso)
```

Comment:
I would remove the "(penso)" word 👀 
Te le faccio pure in inglese le review, toh